fixtures/source-maps/sku.config.ts

```diff
@@ -0,0 +1,12 @@
+import { makeStableHashes } from '@sku-private/test-utils';
+import type { SkuConfig } from 'sku';
+
+export default {
+  clientEntry: 'src/client.tsx',
+  renderEntry: 'src/render.tsx',
+  publicPath: '/static/source-maps',
+  port: 8303,
+  target: 'dist',
+  sourceMapsProd: process.env.SKU_SOURCE_MAPS_PROD === 'true',
```

Comment:
Decided to make this configurable via an envar to avoid having to define 4 different configs for the `source-maps` tests: 1 for each combination of `vite/webpack` and `sourceMapsProd: true/false`.